src/SFA.DAS.EmployerFeedback.Web/Views/Shared/_GoogleAnalyticsBodyPartial.cshtml

```diff
@@ -0,0 +1,5 @@
+﻿<!-- Google Tag Manager (noscript) -->
```

Comment:
Please double-check if this partial view is being used. I’m not sure if this is dynamically referenced. If not, consider removing.